packages/advanced-logic/src/extensions/payment-network/erc20/fee-proxy-contract.ts

```diff
@@ -23,7 +23,7 @@ const erc20FeeProxyContract: ExtensionTypes.PnFeeReferenceBased.IFeeReferenceBas
   isValidAddress,
 };
 
-const supportedNetworks = ['mainnet', 'rinkeby', 'private'];
+const supportedNetworks = ['mainnet', 'rinkeby', 'private', 'matic'];
```

Comment:
in packages/smart-contracts/src/lib/artifacts/ERC20FeeProxy/index.ts there is already `mumbai` defined so will it be added here later ?